app/assets/javascripts/admins/components/CodeCsvCard.js

```diff
@@ -37,8 +37,8 @@ class CodeCsvCard extends React.Component {
     var created_at = new Date(this.props.code_csv.created_at)
     created_at = created_at.toUTCString()
     return (
-      <div className='flex code-csv-card' onClick={this.openCsvModal}>
-        <div className='flex name-date-wrapper'>
+      <div className='flex code-csv-card'>
+        <div className='flex name-date-wrapper' onClick={this.openCsvModal}>
           <div className='code-csv-name'>
```

Comment:
Actually I looked into this a little more and there's a better way to do it. Take a look at `e.stopPropagation()`